genomic_data_service/rnaseq/client.py

```diff
@@ -3,5 +3,9 @@
 
 def add_rna_client(app):
     host = app.config.get('RNA_GET_ES')
-    es = Elasticsearch(host)
+    es = Elasticsearch(
+        host,
+        timeout=200,
```

Comment:
this is a pretty large timeout, any issues with ES?

Reply:
Yeah, can probably reduce it. The default of `10` probably too short for first calculation of some expensive aggregations (ES does some agg caching on subsequent requests).